workshops/rock_paper_scissors/README.md

```diff
@@ -0,0 +1,274 @@
+
+player = False
+
+name = input("Enter your name:")
+print("Type 'reset' to reset score")
+
+You = 0
+PC = 0
+```
+
+### 3)Keeping the game in a loop
+![Image](https://cloud-nmra250be.vercel.app/loop.gif)
+
+For this game I will using `while` loop but you can also use `for` loop! 
+
+So for starters type,
+```
+While Player == False: 
+```
+Now we let the player choose either `Rock` `Paper` or `Scissors`
+
+There are 5 possibilities:
+1. The player chooses `Rock`
+2. The player chooses `Paper`
+3. The player chooses `Scissors`
+4. The word entered is invalid
+5. The player enters `reset`
+
+### 4) Producing separate outcomes for all possibilities
+
+We are going to use if elif & else statement to produce separate outcomes for all possibilities!
+
+Enter the following code:
+```
+while player == False:
+    player = input("Rock, Paper, Scissors?(r,p,s)")
+    if player == computer:
+        print("Tie!")
+        print('Computer =',PC)
+        print(name,'=',You)
+    elif player == "r":
+        if computer == "p":
+            print("You lose!","Paper covers Rock")
+            PC+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+        else:
+            print("You win!","Rock smashes Scissors")
+            You+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+    elif player == "p":
+        if computer == "s":
+            print("You lose!", "Scissors cut Paper")
+            PC+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+        else:
+            print("You win!","Paper covers Rock")
+            You+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+    elif player == "s":
+        if computer == "r":
+            print("You lose...","Rock smashes Scissors")
+            PC+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+        else:
+            print("You win!", "Scissors cuts Paper")
+            You+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+    elif player == "reset":
+      You=1*0
+      PC=1*0
+      print("The score has been reset!")
+    else:
+        print("That's not a valid play. Please select a valid option!")
+```
+#### Explanation:
+
+1. The first if statement declares a tie.
+2. In the first three elif statement, the player can either win or lose and the points are given respectively.
+3. In the fourth elif statement, the player could reset the score.
+4. Last else statement occurs only when the player enters a invalid word.
+
+### 5) Keeping the loop going
+Now as we have completed codin the game we need it to be looped infinite number of times.
+
+To do this you just need to add the following statement after the `while` statement,
+```
+player = False
+```
+This will instantly trigger the loop and keep the game going.
+
+And at last, you need to add the following command so the computer chooses a different outcome every time
+```
+computer = t[randint(0,2)]
```

Comment:
this was defined previously too right?

Reply:
but this wasn't in al loop so if we remove this one the computer will choose only one thing every time (you can remove it and see)

Reply:
No problem at all!